app/src/main/java/com/teamltt/carcare/database/DbHelper.java

```diff
@@ -238,6 +314,81 @@ public int updateVehicle(long vehicleId, Vehicle vehicle) {
         return numAffected;
     }
 
+    public long createNewReminder(long vehicleId, String name, int featureId, int comparison, int value, String date)  {
```

Comment:
`DbHelper` methods should take in models when possible. Convert this method to use the new `Reminder` model.